themes/beaver/assets/css/utilities/flexbox.css

```
@@ -0,0 +1,41 @@
+/* Flexbox Utility Patterns - Extracted from fl-homepage-layout.css */
+
+/* Cross-browser flexbox display */
+.u-flex {
+  display: -ms-flexbox;
+  display: -webkit-flex;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

display is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)